airbyte-integrations/connectors/source-coin-api/source_coin_api/coin_api.yaml

```
@@ -0,0 +1,67 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "{{ 'https://rest.coinapi.io/v1' if config['environment'] == 'production' else 'https://rest-sandbox.coinapi.io/v1' }}"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: "X-CoinAPI-Key"
+      api_token: "{{ config['api_key'] }}"
+    request_options_provider:
+      request_parameters:
+        period_id: "{{ config['period'] }}"
+        time_start: "{{ config['start_time'] }}"
+        time_end: "{{ config['end_time'] }}"
```

```suggestion
        time_end: "{{ config.get("end_time", now_utc()) }}"
```
not completly sure if works if not `config.get("end_time") or now_utc()`

---

It works even if `end_date` is not provided.